src/codeu/chat/client/android/app/src/main/java/com/google/codeu/chatme/view/adapter/ChatListAdapterView.java

```diff
@@ -0,0 +1,20 @@
+package com.google.codeu.chatme.view.adapter;
+
+import com.google.codeu.chatme.model.Conversation;
+
+import java.util.ArrayList;
+
+/**
+ * An interface to handle presenter-delegated actions in order to
+ * update list of conversations in {@link ChatListAdapter}
+ */
+public interface ChatListAdapterView {
+
+    /**
+     * Resets the list of conversations in {@link ChatListAdapter}
+     *
+     * @param conversations new list of conversations
+     */
+    public void setChatList(ArrayList<Conversation> conversations);
```

Comment:
Generally I prefer more abstract types in interface definitions.  Could this be declared as ``List`` or ``Iterable`` or ``Collection`` instead of ``ArrayList``?